app/controllers/daily_report_comments_controller.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+class DailyReportCommentsController < ActionController::Base
```

Comment:
__[brakeman]__
* 'protect_from_forgery' should be called in DailyReportCommentsController [:ref](http://brakemanscanner.org/docs/warning_types/cross-site_request_forgery/)

<!-- Comment by [SideCI](https://sideci.com) -->